pom.xml

```diff
@@ -9,7 +9,7 @@
 
 	<groupId>com.apigee.edge.config</groupId>
 	<artifactId>apigee-config-maven-plugin</artifactId>
-	<version>1.3.6-SNAPSHOT</version>
+	<version>1.3.6</version>
```

Comment:
Should be 1.3.6-SNAPSHOT

Reply:
This is taken care of now. Thanks
@ssvaidyanathan